econml/metalearners/_metalearners.py

```diff
@@ -366,6 +370,7 @@ def __init__(self, *,
         self.models = clone(models, safe=False)
         self.final_models = clone(final_models, safe=False)
         self.propensity_model = clone(propensity_model, safe=False)
+        self.categories = categories
         if categories != 'auto':
```

Comment:
We have to move this logic to fit in this PR

Reply:
Will do